### PR TITLE
{Telemetry} Add command init and invoke time in telemetry

### PR DESCRIPTION
**Pull request by fengzhou-msft**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Telemetry

File: src/azure-cli-core/azure/cli/core/telemetry.py [hunk 1]
```diff
@@ -45,6 +45,8 @@ def __init__(self, correlation_id=None, application=None):
         self.extension_management_detail = None
         self.raw_command = None
         self.mode = 'default'
+        self.init_time_elapsed = None
+        self.invoke_time_elapsd = None
```
Comment (fengzhou-msft) [inline src/azure-cli-core/azure/cli/core/telemetry.py:49]:
```suggestion
        self.invoke_time_elapsed = None
```

**Comment by yonzhan** [comment]
Telemetry

File: src/azure-cli/azure/cli/__main__.py [hunk 2]
```diff
@@ -71,3 +69,7 @@ def cli_main(cli, args):
                     invoke_finish_time - init_finish_time)
     except NameError:
         pass
+
+    telemetry.set_init_time_elapsed("{:.6f}".format(init_finish_time - start_time))
+    telemetry.set_invoke_time_elapsed("{:.6f}".format(invoke_finish_time - init_finish_time))
```
Comment (houk-ms) [inline src/azure-cli/azure/cli/__main__.py:74]: Is it better if we only record the `init_finish_time` to telemetry?

Now that we alreay have `start_time`, `init_finish_time` and `end_time`, it seems not necessary here to introduce anthor `invoke_finish_time`.

Comment (fengzhou-msft) [reply src/azure-cli/azure/cli/__main__.py:74]: For convenience and consistency purpose, `start_time`, `init_finish_time`, `invoke_finish_time` are all generated with `timeit.default_timer()` while the `start_time` and `end_time` in telemetry session use `datetime.datetime.utcnow()`.

File: .flake8 [hunk 3]
```diff
@@ -2,6 +2,7 @@
 max-line-length = 120
 max-complexity = 10
 ignore =
+    E402,  # module level import not at top of file
```
Comment (haroldrandom) [inline .flake8:5]: Is it able to disable at the line-level to minimize the impact?

Comment (fengzhou-msft) [reply .flake8]: updated

**Pull request merged**